README.md

```diff
@@ -43,6 +41,10 @@ run_NaroNet(path,params)
 get_BioInsights(path,params)
 ```
 
+The path value should be set to 'DATASET_DATA_DIR' as shown in line 83 from [file](https://github.com/CarolRameder/NaroNet/blob/main/NaroNet-main/src/main.py).
+
+# Exp 1
```

Comment:
Can you add a description of what you're trying to do in each experiment? You can just copy what you have from the thesis.

Reply:
More details than the list at the start of the file? (Lines 5, 6, 7)

Reply:
If you mention what you listed again in the headers it's fine